src/test/pia-format.test.ts

```diff
@@ -87,6 +120,47 @@ const sample20piaSimplified = `01          06221952
 25       0.00       0.00       0.00       0.00       0.00       0.00   20000.00  104400.00  105480.00  108000.00
 26  113040.00  117000.00  122400.00  128160.00   35000.00`;
 
+const sample25piaSimplified = `01          06221952
+031072014
+0619812019
+12   1500.00
+22    2000.00    2000.00    2000.00    2000.00    2000.00    2000.00    2000.00    2000.00    2000.00    2000.00
+23    4000.00    4000.00    4000.00    4000.00    4000.00    4000.00    4000.00    4000.00    4000.00    4000.00
+24    6000.00    6000.00    6000.00    6000.00    6000.00    6000.00    6000.00    6000.00    6000.00    6000.00
+25    8000.00    8000.00  113700.00    8000.00    8000.00    8000.00    8000.00    8000.00    8000.00`;
+
+// Generic function for creating full retirement sample PiaFormat
+function createPiaSampleFormat(sampleRetirementValues: any) {
+  //Background finance info for sample
+  const earnings =
+    sampleRetirementValues["osss:OnlineSocialSecurityStatementData"][
+    "osss:EarningsRecord"
+    ]["osss:Earnings"];
+  
+  const userDOB = dayjs("1952-06-22").toDate();
+  const userDOR = dayjs("2014-07-22").toDate(); // 66yo is their full retirement age.
```

Comment:
Can you please parameterize these and the user pension and pass them in according to the links below? 

I forgot to give you the different parameters for sample 25 and 1 and 2. They're slightly different:
- sample 1: http://thadk.net/anypiamac-docs/html/Samples/sample_1.html
- sample 2: http://thadk.net/anypiamac-docs/html/Samples/sample_2.html
- sample 25 (despite URL): http://thadk.net/anypiamac-docs/html/Samples/sample_27.html

here's all of the samples (in case you need it): http://thadk.net/anypiamac-docs/html/